File: src/main/java/dev/infernity/polarisplugin/polarisplugin/commands/dieCommand.java
package dev.infernity.polarisplugin.polarisplugin.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class dieCommand implements CommandExecutor {
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (sender instanceof Player) {

            Player p = (Player) sender;

            p.setHealth(0f);

        }
        return false;
    }
}

File: src/main/java/dev/infernity/polarisplugin/polarisplugin/commands/dieCommand.kt
package dev.infernity.polarisplugin.polarisplugin.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class dieCommand : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (sender is Player) {
            sender.health = 0.0
        }
        return false
    }
}